configure.ac

```diff
@@ -165,8 +165,8 @@ AS_IF([test x"$enable_hardening" != x"no"], [
   add_hardened_c_flag([-Wstrict-overflow=5])
   add_hardened_c_flag([-Wbool-compare])
 
-  add_hardened_define_flag([-U_FORTIFY_SOURCE])
-  add_hardened_define_flag([-D_FORTIFY_SOURCE=2])
+  CFLAGS="${CFLAGS} -O2"
```

Comment:
add_hardened_c_flag([-O2])

Reply:
Indeed, that makes much more sense, thanks!